backend/src/main/kotlin/no/nav/tilbakekreving/burdeforstatt/App.kt

```diff
@@ -37,41 +42,61 @@ fun main() {
     )
 
     val httpClient = defaultHttpClient()
-    val config = Config(
+    val appConfig = AppConfig(
         tokenEndpoint = System.getenv("NAIS_TOKEN_ENDPOINT"),
         tokenExchangeEndpoint = System.getenv("NAIS_TOKEN_EXCHANGE_ENDPOINT"),
         tokenIntrospectionEndpoint = System.getenv("NAIS_TOKEN_INTROSPECTION_ENDPOINT")
     )
 
+    val mqConfig = MqConfig(
+        host = "b27apvl220.preprod.local",
+        port = 1413,
+        channel = "Q1_FAMILIE_TILBAKE",
+        queueManager = "MQLS02",
+        queue = "QA.Q1_FAMILIE_TILBAKE.KRAVGRUNNLAG",
+        user = System.getenv("CREDENTIAL_USERNAME"),
+        password = System.getenv("CREDENTIAL_PASSWORD"),
+    )
+
+    val mqService = MQService(mqConfig)
+
     embeddedServer(Netty, port = 8080) {
-        registerApiRoutes(config, httpClient)
+        registerApiRoutes(appConfig, httpClient, mqService)
     }.start(wait = true)
 
 }
 
-private fun Application.registerApiRoutes(config: Config, httpClient: HttpClient) {
+private fun Application.registerApiRoutes(appConfig: AppConfig, httpClient: HttpClient, mqService: MQService) {
     val authClient = AuthClient(
-        config = config,
+        appConfig = appConfig,
         httpClient = httpClient
     )
     authentication {
         register(TexasAuthenticationProvider(TexasAuthenticationProvider.Config("azuread", authClient)))
     }
 
+    var tilbakekrevingUrl = "https://tilbakekreving-backend.intern.nav.no/api/behandling/v1"
```

Comment:
Kan bruke service for API kall. `http://tilbakekreving-backend`. Slipper å gå via internett og går heller direkte innad i cluster